src/EnergyPlus/ZoneAirLoopEquipmentManager.cc

```diff
@@ -530,6 +521,16 @@ namespace ZoneAirLoopEquipmentManager {
 			EachOnceFlag( AirDistUnitNum ) = false;
 		}
 
+		if ( AirDistUnit( AirDistUnitNum ).ZoneNum != 0 && FirstHVACIteration && DataHeatBalance::Zone( AirDistUnit( AirDistUnitNum ).ZoneNum ).HasAdjustedReturnTempByITE ) {
+			for ( int AirDistCompNum = 1; AirDistCompNum <= AirDistUnit( AirDistUnitNum ).NumComponents; ++AirDistCompNum ) {
+				if ( AirDistUnit( AirDistUnitNum ).EquipType_Num( AirDistCompNum ) != SingleDuctVAVReheat && AirDistUnit( AirDistUnitNum ).EquipType_Num( AirDistCompNum ) != SingleDuctVAVNoReheat ) {
+					ShowSevereError( "The FlowControlWithApproachTemperatures only applies to ITE zones with single duct VAV terminal unit." );
+					ShowContinueError( "The return air temperature of the ITE will not be overwritten." );
```

Comment:
Since the next line is a fatal error, this line is not needed.
In the line above I would change the wording from "only applies to . . . " to "only works with  . . . ".

Placed here, this check will execute every time step, looping through all AirDistUnits. This should move up into the EachOnceFlag block above, and it doesn't need to loop through the AirDistUnits, Just check if the zone has HasAdjustedReturnTempByITE  and the terminal unit type is not single duct VAV (reheat or no reheat).
Also, don't need to pass in FirstHVACIteration.

Reply:
@Myoldmopar OK thanks. I misread the loop, but it's still unnecessary to even do this once every time step in every run with air distribution units.  I can fix that next after I get the coil sizing stuff up.

Reply:
That's up to you.  This isn't **your** pull request to be fixing at the last minute...

Reply:
Thanks @Myoldmopar for cleaning up the doc issue. I fixed this accordingly.